frontend/src/pages/PlacePage.vue

```diff
@@ -126,6 +126,13 @@ export default {
 };
 </script>
 
+<style lang="scss">
+    .place-top-info .rating-wrapper .rating {
```

Comment:
no, such things aren't acceptable. if you need to customize rating compoent do it on profile page by adding another css class, but not this stuff )